platform/linux-generic/odp_packet.c

```diff
@@ -1982,12 +1983,31 @@ static inline void parse_tcp(packet_parser_t *prs,
 	*parseptr += (uint32_t)tcp->hl * 4;
 }
 
+static inline void parse_vxlan(packet_parser_t *prs, packet_parser_t *in_prs,
+			       const uint8_t **parseptr, uint32_t *offset,
+			       uint32_t frame_len, uint32_t seg_len)
+{
+	const _odp_vxlanhdr_t *vxlan = (const _odp_vxlanhdr_t *)*parseptr;
+
+	if (ODPH_VXLAN_BIT(vxlan->flags))
```

Comment:
do not use helper in platform code. Theoretically that should not even compile. (I hope I will add automatic check for that some day.)

Reply:
@muvarov @psavol 
This issue keeps on risiging again and again. Maybe we can have odp-proto that will be used by odp-helper and by linux-generic (and any other implementation that will need it) but will not be part of ODP API? Code/declaration duplication between linux-generic protocols subdir and helper protocols support is really painful.